docs/articles/documentation/test-api/built-in-waiting-mechanisms.md

```diff
@@ -10,9 +10,15 @@ redirect_from:
 TestCafe has built-in automatic waiting mechanisms, so that it does not need dedicated API to wait for page elements to appear or redirects to happen.
```

Comment:
TestCafe has built-in automatic waiting mechanism and does not require dedicated API to wait for redirects or page elements to appear.